src/applications/caregivers/config/form.js

```diff
@@ -0,0 +1,360 @@
+  chapters: {
+    veteranChapter: {
+      title: 'VETERAN/SERVICE MEMBER',
+      pages: {
+        veteranInfoOne: {
+          path: 'service-member',
+          title: 'Veteran Information',
+          uiSchema: {
+            'ui:description': VetInfo,
+            [vetFields.fullName]: fullNameUI,
+            [vetFields.ssn]: vetUI.ssnUI,
+            [vetFields.dateOfBirth]: dateOfBirthUI,
+            [vetFields.gender]: genderUI,
+          },
+          schema: {
+            type: 'object',
+            required: [vetFields.fullName, vetFields.dateOfBirth],
+            properties: {
+              [vetFields.fullName]: veteranProps.fullName,
+              [vetFields.ssn]: veteranProps.ssnOrTin,
+              [vetFields.dateOfBirth]: veteranProps.dateOfBirth,
+              [vetFields.gender]: veteranProps.gender,
+            },
+          },
+        },
+        veteranInfoTwo: {
+          path: 'service-member-page2',
+          title: 'Veteran Information (Continued)',
+          uiSchema: {
+            'ui:description': VetInfo,
+            [vetFields.address]: addressUI,
+            [vetFields.primaryPhoneNumber]: primaryPhoneNumberUI,
+            [vetFields.alternativePhoneNumber]: alternativePhoneNumberUI,
+            [vetFields.email]: emailUI,
+            [vetFields.vaEnrolled]: vetUI.vaEnrolledUI,
+            [vetFields.plannedClinic]: vetUI.plannedClinicUI,
+            [vetFields.facilityType]: facilityTypeUI,
+            [vetFields.previousTreatmentFacility]: previousTreatmentFacilityUI,
+          },
+          schema: {
+            type: 'object',
+            required: [],
+            properties: {
+              [vetFields.address]: getAddressSchema.schema(fullSchema, true),
+              [vetFields.primaryPhoneNumber]: phone,
+              [vetFields.alternativePhoneNumber]: phone,
+              [vetFields.email]: veteranProps.email,
+              [vetFields.vaEnrolled]: veteranProps.vaEnrolled,
+              [vetFields.plannedClinic]: veteranProps.plannedClinic,
+              [vetFields.previousTreatmentFacility]:
+                veteranProps.lastTreatmentFacility,
+            },
+          },
+        },
+      },
+    },
+    primaryCaregiverChapter: {
+      title: 'PRIMARY FAMILY CAREGIVER',
+      pages: {
+        primaryCaregiverInfoOne: {
+          path: 'primary-caregiver-page1',
+          title: 'Primary Caregiver Information',
+          uiSchema: {
+            'ui:description': () =>
+              PrimaryCaregiverInfo({ additionalInfo: true }),
+            [primaryCaregiverFields.fullName]: fullNameUI,
+            [primaryCaregiverFields.ssn]: primaryCaregiverUI.ssnUI,
+            [primaryCaregiverFields.dateOfBirth]: dateOfBirthUI,
+            [primaryCaregiverFields.gender]: genderUI,
+          },
+          schema: {
+            type: 'object',
+            required: [
+              primaryCaregiverFields.fullName,
+              primaryCaregiverFields.dateOfBirth,
```

Comment:
Should the SSN be required? I would think you get paid for being designated as a primary caregiver.